packages/esm-patient-vitals-app/src/vitals-biometrics-form/vitals-biometrics-form.test.tsx

```diff
@@ -38,9 +44,13 @@ jest.mock('../common', () => ({
   })),
 }));
 
+mockUseConfig.mockReturnValue({
+  ...getDefaultsFromConfigSchema(configSchema),
+  mockVitalsConfig,
```

Comment:
This seems like it would create a key called `mockVitalsConfig` on the object, which I would really hope would produce a type error

Reply:
Fixed in the latest push, I think.